lib/src/base/mixins/network_list.dart

```diff
@@ -134,6 +177,9 @@ mixin NetworkListBaseMixin<T, S extends NetworkListStateBase<T>>
     );
   }
 
+  /// Use [editItemAsync] to edit item in the `List` asynchronously.
+  ///
+  /// For this method to work properly, the [equals] MUST be overridden in the [NetworkListCubit] or [NetworkListBloc]. When extending [NetworkListCubit] or [NetworkListBloc], the IDE will warn that this method requires an override due to the missing implementation.
   FutureOr<void> editItemAsync(T updatedItem) async {
```

Comment:
No need to write about `equals` cause  in required to override

Reply:
For other methods, you can simply specify that the `equals` are used in this method